### ISSUE TITLE
Accessibility: In top-level menus, menuitems not contained by proper parent 

### ISSUE THREAD

**Issue by dgodongm** [mastodon/joinmastodon#363]
For the Resources and Language top-level menus, the list items are set with role as "menuitem" but their parent is missing its proper role of "menu". This will help assistive technology guide users on navigating these menus. 
Refer to https://dequeuniversity.com/rules/axe/4.7/aria-required-parent?application=AxeChrome and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/menuitem_role

See axe screenshot below for example:
![Screenshot 2023-06-27 at 2 30 52 PM](https://github.com/mastodon/joinmastodon/assets/123777263/5ac42a99-1b7c-4148-964e-6edbc270a2f8)
